docker/docker-compose.yml

```diff
@@ -11,11 +11,15 @@ services:
     volumes:
       - /var/lib/mysql
```

Comment:
I think this should be removed - there's no need to share /var/lib/mysql from the host machine anymore, is there?